tests/functional/tests/tests.yaml

```diff
@@ -1,10 +1,11 @@
 charm_name: nrpe
 gate_bundles:
-  - noble
-  - jammy
+  # - noble
+  # - jammy
   - focal
 smoke_bundles:
-  - jammy
+  - focal
+  # - jammy
```

Comment:
TODO: revert this when the ubuntu charm pushes releases supporting noble and jammy on s390x and ppc64el.